docs/modules/ROOT/pages/vshn-managed/postgresql/usermanagement.adoc

```diff
@@ -65,15 +69,21 @@ spec:
   writeConnectionSecretToRef:
     name: postgres-creds
 ----
-<1> Removing this entry would remove the `app1` user and the grant giving it access to the app1 database. It will not remove the database, as it's still referenced by user app2.
-<2> Removing this entry would remove the user `app2` and the grants giving access to database app1. It will not remove the database app1, as it's still referenced by user `app1`
-<3> Removing this entry will remove the user `app3` and its database.
+<1> Removing this entry will stop managing the `app1` user, but the user and database will remain in PostgreSQL
+<2> Removing this entry will stop managing the user `app2`, but the user and its grants will remain in PostgreSQL
+<3> Removing this entry will stop managing the user `app3`, but the user and database will remain in PostgreSQL
 
-[IMPORTANT]
-.Non-reversible operation
+To manually clean up users, databases, and grants that are no longer managed:
+
+1. Connect to your PostgreSQL instance using the connection details
+2. Drop the user: `DROP USER username;`
+3. Drop the database (if no longer needed): `DROP DATABASE databasename;`
+4. Revoke grants if needed: `REVOKE ALL PRIVILEGES ON DATABASE databasename FROM username;`
```

Comment:
I guess PostgreSQL allows this, but logically speaking the grant depends on the database and the user, so deleting it last might not make sense.